### ISSUE TITLE
Move individual menu management to the menu list view tab (or more intuitive location)

### ISSUE THREAD

**Issue by ndiego** [WordPress/gutenberg#51536]
## What problem does this address?
I have fielded numerous questions about how to rename Navigation menus or delete them once created. The most recent in the latest [call for testing](https://make.wordpress.org/test/2023/06/07/fse-program-testing-call-24-momery-makeover/). This functionality is available but hidden in the Advanced panel with the Settings tab. This is not intuitive, and most users will never find it. 

## What is your proposed solution?
Move all menu settings to the List View tab. 

| Current | Proposed Change |
| --- | --- |
| <img width="811" alt="image" src="https://github.com/WordPress/gutenberg/assets/4832319/043d6b8b-f608-4c1b-805b-06cd5404852a"> | <img width="972" alt="image" src="https://github.com/WordPress/gutenberg/assets/4832319/156c6282-354e-4c21-a8b3-c89b1bb0b334"> |

**Comment by ndiego** [comment]
@richtabor @getdave would love your thoughts on this.

**Comment by getdave** [comment]
Navigation in Browse mode helps with this. We could have a modal popup as an action under the ellipsis menu and that would be consistent with what we have in Browse Mode sidebar.

**Comment by ndiego** [comment]
> Navigation in Browse mode helps with this. We could have a modal popup as an action under the ellipsis menu and that would be consistent with what we have in Browse Mode sidebar.

Yeah that would work great too. Really anything to make this functionality easier to access. 😅

**Comment by annezazu** [comment]
Noting this option: https://github.com/WordPress/gutenberg/issues/50583 I agree that we should have consistency across the experiences though as this came up previously: https://github.com/WordPress/gutenberg/issues/46638

**Comment by getdave** [comment]
I recognise that this needs to be improved. However I"m not sure about putting this information in the `List View` tab which should probably be reserved for tree presentation only.

I did consider `Settings` as a home, but it would end up further down the panel because (AFAIK) there is no way to specify the _order_ of panel items and other controls are inserted via block supports and thus end up at the top.

I'd appreciate some additional input from @WordPress/gutenberg-design before proceeding here. Also pinging @scruffian and @draganescu for their thoughts.

**Comment by scruffian** [comment]
I think we should take inspiration from the site view:
<img width="538" alt="Screenshot 2023-09-08 at 14 20 33" src="https://github.com/WordPress/gutenberg/assets/275961/f95e1074-9bc9-4256-b6fd-6b804152790d">

**Comment by richtabor** [comment]
Why not use the "Rename" flow we have in place for Group blocks? I suspect that'll become an expected pattern in time.

<img width="600" alt="CleanShot 2023-09-08 at 17 27 18" src="https://github.com/WordPress/gutenberg/assets/1813435/94a3d9f5-b3c3-4a99-a3d0-37c2611ce1be">

**Comment by getdave** [comment]
I think some combination of Rich and Ben's suggest seems logical. I don't think _relying_ on List View will suffice as the control should be contextual to the block when it is selected. We can add that as an additional affordance though.

If we were able to [move the menu _switching_ out of the 3 dots menu](https://github.com/WordPress/gutenberg/pull/54214) on the List View tab then perhaps we could add `Rename` there and that would mirror the pattern already found in the Site View sidebar as suggested by @scruffian. 

I don't think however, that we can simp;y dump `Rename` into the existing 3-dots options menu because it's currently clearly as menu _switcher_ for the Navigation _Block_ rather than an options menu for the current Navigation _Menu_.
